### PR TITLE
Portfolio created

### PR DESCRIPTION
**Pull request by AirwreckEye**
I wasn't sure when to commit and ended up coding everything at once. I apologize for the inconvenience.

### CODE REVIEW

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  
+
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/about.html:12]: Small nit: clean up any empty lines

File: portfolio/src/main/webapp/about.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  </head>
+  <body>
+     
+      <h1 class="jumbotron"> <center> About Me</center> </h1>
+    <nav>
+    <ul>
+  <li><a href="index.html">Home</a></li>
+  <li><a href="about.html">About</a></li>
+  <li><a href="gallery.html">Gallery</a></li>
+  <li><a href="poems.html"> Favorite Poems</a></li>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/about.html:22]: nit: remove empty space here and in the next one

File: portfolio/src/main/webapp/about.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/about.html:23]: XD nice

File: portfolio/src/main/webapp/about.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        I speak 3 languages - English, Igbo, and Spanish
+
+        </h3>
+     </p>
+
+    </div>
+    <!-- Javascript for Bootstrap -->
+  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/about.html:46]: Usually these belong at the top of the html file (kind of like imports)

File: portfolio/src/main/webapp/gallery.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+<nav>
+      <ul>
+   <li><a href="index.html">Home</a></li>
+  <li><a href="about.html">About</a></li>
+  <li><a href="gallery.html">Gallery</a></li>
+  <li><a href="poems.html"> Favorite Poems</a></li>
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
+      </ul>
+</nav>
+
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/gallery.html:24]: If this is pasted in each file, is it possible to have this be in a separate file that you import?

File: portfolio/src/main/webapp/gallery.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  <li><a href="gallery.html">Gallery</a></li>
+  <li><a href="poems.html"> Favorite Poems</a></li>
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
+      </ul>
+</nav>
+      
+      
+          
+       <div class="container">
+      <img src="media/portfolioimage.jpg" alt="Eric on a bridge." class="img-circle" style="width:410px;height:410px;">
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/gallery.html:28]: I don't see any of these image files in your pull request, you'll need to "git add" them as well

File: portfolio/src/main/webapp/gallery.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+        
+
+
+
+      
+
+    <!-- Javascript for Bootstrap -->
+  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/gallery.html:68]: Same here with common imports, you can create a script.js file and just add these to that file and each of your html files can just import script.js

File: portfolio/src/main/webapp/poems.html [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
+    </ul>
+    </nav>
+       <div class="container">
+
+
+      <p>
+       
+
+ <h3>Ka’Ba
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/poems.html:32]: very profound poems

File: portfolio/src/main/webapp/topten.html [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    </ul>
+      </nav>
+      
+       <div class="container">
+
+
+      
+        <h3>
+        <ol>
+            <li>FullMetal Alchemist(Series)</li>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/topten.html:34]: Oldschool or Brotherhood?

File: portfolio/src/main/webapp/topten.html [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+            <li>Erased (Series)</li>
+            <li>Your Name/ Kimi No Na Wa (Film)</li>
+            <li>Weathering With You (Film)</li>
```
Comment (rharagutchi) [inline portfolio/src/main/webapp/topten.html:37]: Curious where you'll rank 5cm/s since they're all Makoto Shinkai movies :)

**Review by rharagutchi — COMMENTED** [review]
Looks great so far! Mostly just nitpicking

File: portfolio/src/main/webapp/about.html [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  
+
+  </head>
+  <body>
+     
+      <h1 class="jumbotron"> <center> About Me</center> </h1>
```
Comment (codewithjess) [inline portfolio/src/main/webapp/about.html:16]: The indentation starting from this line is a little off for this file and others. Please see https://google.github.io/styleguide/htmlcssguide.html#Indentation

Each nested element should be indented only once.

**Review by codewithjess — COMMENTED** [review]
Nice job, I like the content of your portfolio! Just added a comment about the indentation of your files. Proper indentation and spacing can make code more readable :)

File: portfolio/src/main/webapp/about.html (hunk 3 above)
Comment (AirwreckEye) [reply portfolio/src/main/webapp/about.html:23]: Thanks hahah

**Review by rharagutchi — APPROVED** [review]

**Pull request merged**